docs/license.rst

```diff
@@ -0,0 +1,9 @@
+License
+=======
+
+.. include:: ../AUTHORS.rst
+
+.. note::
+    In applying this license, CERN does not waive the privileges and immunities
+    granted to it by virtue of its status as an Intergovernmental Organization or
+    submit itself to any jurisdiction.
```

Comment:
As for the copyright, I think it's import to assert whether the license is something CERN should decide on, or the AAS/Sloan.